### DISCUSSION TITLE
More communication channels for community (slack?)

### DISCUSSION THREAD

**Discussion by twodrops** [useblocks/sphinx-needs#207] [Ideas]
At the moment, the discussion on sphinx-needs is limited to GitHub Issues. It would be great to have a communication channel like Slack to exchange between sphinx-needs developers + users and among users themselves.
The easiest would be to use the WriteTheDocs -> Sphinx Channel.

As part of this issue, the communication and invitation to the (decided) channel needs to be published in visible places of sphinx-needs social media.

https://www.writethedocs.org/slack/

![2021-02-15_16h38_37](https://user-images.githubusercontent.com/103930/107967431-62a5f200-6fad-11eb-9a63-020a2e5cdf75.png)

**Comment by danieleades** [comment]
i'm for it. i'm slowing working my way through this codebase in order to have my head around it enough to start fixing some bugs. I have plenty of questions

**Comment by danieleades** [comment]
i wouldn't mind asking some questions about the `services` submodule for example. It's *very* difficult to read

**Comment by danwos** [comment]
I also like the idea to have a more direct communication channel.

However, me wouldn't be (currently) able to join such a channel very often, as my daily work and the private life take a lot of time. Also some of us live in different time zones, so the chance to meet somebody in the channel may not be so big (for our small community).

I'm just thinking of a more asynchronous way, the good old email  group.
I know it's old fashion, but it works and conversations are findable and archived for everybody on the net.
So there is the chance to be helpful also for prospective community members.

I have already created one via google groups: https://groups.google.com/g/sphinx-needs
Content is accessible to everybody on the net and everybody can join without any check from my side.
But only group members can post.

I also would like to suggest a video/voice meeting via zoom or alternatives.
Maybe we could have a "call" once a month and everybody can join.
If topics are huge or urgent, I'm also open for more frequent meetings or 1:1 meetings.

So what do you thing?

Just a reminder, open tasks:

- [ ] Mail group link in the docs
- [ ] Repeating meeting in the docs (if you like it)
- [ ]  1-2 additional admins for the group

**Comment by danieleades** [comment]
just joined the google group. have not used it before, but it seems a good solution

**Comment by danwos** [comment]
It's the same service/mechanism how the sphinx and sphinx-dev user groups are working.
Stole the idea from there ;)

**Comment by twodrops** [comment]
I find the idea good. Do we need to split between sphinx-needs and sphinx-needs-dev or keep it together for now?

Let's publish the group and start talking there :)

**Comment by twodrops** [comment]
First post is available 😃  https://groups.google.com/g/sphinx-needs/c/7UGzcLMtYsw

Btw, https://www.reddit.com/ is also an option. Seems to be more active than google groups. Has nice Mobile Apps as well. 
I like it more than Google Groups.

What do you say?

**Comment by danieleades** [comment]
I don't have a dog in this race. Google groups has the advantage of already being implemented.

**Comment by twodrops** [comment]
I somehow am not happy with Google Groups :( The easiest is if @danwos could turn on the "Discussions" in GitHub Repo settings. Then we can have the forum directly here.
![2021-02-24_12h40_04](https://user-images.githubusercontent.com/103930/108995509-7e8c5080-769d-11eb-840f-617984aea332.png)
![2021-02-24_12h03_57](https://user-images.githubusercontent.com/103930/108995514-80eeaa80-769d-11eb-8707-aa2bc52b7539.png)

**Comment by danwos** [comment]
Ohh nice idea, didn't know github has something like "discussion".
Activated and free to go!

And also transformed this issue to a discussion for a first real-life test :)

**Comment by danwos** [comment]
After moving 1-2 issues and writing 1-2 answers, I'm already very happy with the "discussion" solution.
Love it that everything is at one place and can be linked together.

I would say we found our solution.

**Reply by twodrops** [reply]
Thanks @danwos. I guess once @danieleades copies his introduction from Google Groups to https://github.com/useblocks/sphinxcontrib-needs/discussions/206, we can deactivate the google groups.
Btw, I will add my intro soon :)
